client/asset/btc/btc_test.go

```diff
@@ -225,9 +225,7 @@ func (c *tRPCClient) GetRawTransactionVerbose(txHash *chainhash.Hash) (*btcjson.
 	return c.mpVerboseTxs[txHash.String()], c.rawVerboseErr
 }
 
-func (c *tRPCClient) Disconnected() bool { return false }
-
-func (c *tRPCClient) RawRequest(method string, params []json.RawMessage) (json.RawMessage, error) {
+func (c *tRPCClient) RawRequest(_ context.Context, method string, params []json.RawMessage) (json.RawMessage, error) {
```

Comment:
Note that all the other `tRPCClient` need to go away and instead have `RawRequest` handle them now.

Reply:
💯